ProjectSource/app-library/src/test/java/com/kayako/sdk/android/k5/common/view/CircleImageViewTest.java

```diff
@@ -0,0 +1,379 @@
+package com.kayako.sdk.android.k5.common.view;
+
+import static junit.framework.Assert.fail;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.powermock.api.support.membermodification.MemberMatcher.method;
+import static org.powermock.api.support.membermodification.MemberModifier.suppress;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.ColorFilter;
+import android.graphics.Paint;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.ImageView.ScaleType;
+import com.kayako.sdk.android.k5.R;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({CircleImageView.class,ImageView.class,Paint.class,View.class})
+public class CircleImageViewTest {
+
+  private final static String INVALID_BOUNDS_MESSAGE = "adjustViewBounds not supported.";
+  private final int padLeft = 5;
```

Comment:
⚠️  This snippet of code contains class fields that have a "constant" behavior - they are initialized at declaration time, are not part of a constructor and are not modified in this class. Consider marking these fields as `static final` to transform them into Java constants and take advantage of the compiler support if we try to intentionally/accidentally modify them.  

**[More details](https://confluence.devfactory.com/x/TwcaF)**
          
## The possible solution in this case
```java
private static final int PAD_LEFT = 5;
``` 

<a id="31d1b210-4e70-11e8-8f04-8143e47b96ef" href="none" codereview-uuid-reserved-tag />